css/local.css

```diff
@@ -35,6 +34,14 @@ body.expand {
     background-color var(--duration) var(--timing);
 }
 
+.top-gradient {
+  background-image:
+      linear-gradient(to bottom, var(--background-gradient-color), transparent);
+  background-size: 100% 2%;
```

Comment:
With the background size being determined by page length, the gradient will look different on pretty much every page.